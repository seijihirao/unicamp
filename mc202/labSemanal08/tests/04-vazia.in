excluir 1
buscar 1
minimo
maximo
pos-ordem
em-ordem
pre-ordem
largura
terminar
