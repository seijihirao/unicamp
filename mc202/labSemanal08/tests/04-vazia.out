nao pertence
vazia
vazia
vazia
vazia
vazia
vazia
